src/app/workspace/search/search-panel/search-panel.component.ts

```diff
@@ -76,10 +68,7 @@ export class SearchPanelComponent implements OnInit {
     ) { }
 
     ngOnInit() {
```

Comment:
do we still need `ngOnInit` here?

Reply:
I think we have it everywhere, even when it's empty. Should I remove all empty angular lifecycle hooks?